packaging/windows/tests/enterprise-module-smoke.ps1

```
@@ -0,0 +1,645 @@
+$programFiles = [Environment]::GetFolderPath([Environment+SpecialFolder]::ProgramFiles)
+$programData = [Environment]::GetFolderPath([Environment+SpecialFolder]::CommonApplicationData)
+$modulePath = [IO.Path]::GetFullPath((Join-Path $PSScriptRoot '..\DefenseClawEnterprise.psm1'))
+$token = ([Guid]::NewGuid().ToString('N')).Substring(0, 10)
+$installRoot = Join-Path $programFiles 'Cisco\DefenseClaw'
+$stateRoot = Join-Path $programData 'Cisco\DefenseClaw'
+$gatewayService = 'DefenseClawGateway'
+$guardianService = 'DefenseClawHookGuardian'
+$customInstallRoot = Join-Path `
+    $programFiles `
+    "Cisco\DefenseClaw-ModuleSmoke-$token\DefenseClaw"
+$customStateRoot = Join-Path `
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'customStateRoot' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)